src/main.nim

```diff
@@ -6485,6 +6511,31 @@ proc handleGlobalKeyEvents(a) =
         openUserManual(a)
 
     # }}}
+    # {{{ emSelectJumpToLink
+    of emSelectJumpToLink:
+      proc handleMoveKey(dir: CardinalDir, mods: set[ModifierKey]; a) =
+        var destIx: int
+        case dir:
+        of dirE, dirN:
+          destIx = ui.jumpToDestinationIx + 1
+        of dirW, dirS:
+          destIx = ui.jumpToDestinationIx - 1
+        ui.jumpToDestinationIx = destIx.floorMod(ui.jumpToDestinations.len)
+        moveCursorTo(ui.jumpToDestinations[ui.jumpToDestinationIx], a)
+        setSelectJumptoLinkActionMessage(a)
+
+
+      handleMoveKeys(ke, allowWasdKeys=true, allowRepeat=false, handleMoveKey)
+
+      if ke.isShortcutDown(scAccept):
+        ui.editMode = emNormal
+        clearStatusMessage(a)
+      elif ke.isShortcutDown(scCancel):
+        moveCursorTo(ui.jumpToSrcLocation, a)
+        ui.editMode = emNormal
+        clearStatusMessage(a)
```

Comment:
This is nice overall, but as I mentioned above, I'd like a different behaviour here that makes the feature more consistent with the single src & dest link case where you can repeatedly press `G` to jump back and forth between the src and dest cells.

We should store the last selected source cell, and pressing `G` in link source select mode should still jump back and forth between the fixed destination cell and the last selected source cell.